frontend/src/streamlist/__tests__/StreamsTable.test.jsx

```diff
@@ -191,10 +191,8 @@ describe("<StreamsTable/> component", () => {
         it(`should have export "${expected[7].export}"`, () => {
           expect(result[7].export).toBe(expected[7].export);
         });
-        it(`should have a render() function that returns a <DeleteStreamSummoner/> component`, () => {
-          const dummyData = {
-            id: 444
-          };
+        it(`should have a render() function that returns a <StreamDetailsButton/> component`, () => {
+          const dummyData = {};
           const renderResult = result[7].render(dummyData);
```

Comment:
Why do we need empty dummyData now?

Reply:
mistake as per other [comment](https://github.com/bean-pod/switchboard/pull/383#discussion_r594005392). fixed 05cb998